### ISSUE TITLE
bug: Touch events are less responsive with iOS 13.1

### ISSUE THREAD

**Issue by gabrielprzb** [ionic-team/ionic#19299]
# Bug Report

**Ionic version:**
[x] **4.9**

**Current behavior:**
Touch events respond less effectively on iOS 13.1, immediately after finishing scrolling, something that did not happen with previous versions of iOS. The user must wait between 0.5-1 seconds after scrolling for a touch event to emit.

**Expected behavior:**
The touch events should fire immediately after finishing scrolling.

**Steps to reproduce:**

1. Create an application with the newest version of ionic and Cordova 5.0.1
2. Create elements and associate them with touch events
3. Run the app on a device that has iOS 13
4. Scroll, stop scrolling, and immediately touch any of the elements that react to some touch event.

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 5.2.7 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.9.0
   @angular-devkit/build-angular : 0.803.3
   @angular-devkit/schematics    : 8.3.3
   @angular/cli                  : 8.3.3
   @ionic/angular-toolkit        : 2.0.0

Cordova:

   Cordova CLI       : 8.0.0
   Cordova Platforms : android 7.0.0, ios 5.0.1
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 3.1.2, (and 17 other plugins)

Utility:

   cordova-res : not installed
   native-run  : not installed

System:

   ios-deploy : 1.9.2
   NodeJS     : v10.10.0 (/usr/local/Cellar/node/10.10.0/bin/node)
   npm        : 6.4.1
   OS         : macOS Mojave
```

**Comment by liamdebeasi** [comment]
Thanks for the issue. I believe this is a bug in the iOS 13.1 beta as I have experienced the same behavior just by using Safari normally. If you are able to reproduce an issue specific to Ionic, feel free to open a new issue.

Thanks!

**Comment by jkronborg** [comment]
I also immediately noticed this after upgrading to iOS 13.0. As Liam says, this is a WebKit/Safari issue. Incoming touch events like `touchstart` are noticeably throttled to 2-3 events per second. Here's a simple demo with a few buttons: https://kronb.org/click.html

I don't have the latest iOS beta handy to see if it's fixed there, but I hope so.

**Comment by jkronborg** [comment]
On iOS 13.1 and 13.1.1 the behavior has changed: `click` events are still 'throttled' (basically the second tap in a double tap is ignored), but e.g. `touchstart` and `touchend` are not.

Good news is that it looks like it has been recently fixed for a coming iOS release: https://trac.webkit.org/changeset/250447/webkit

**Comment by jkronborg** [comment]
This issue has been fixed in iOS 13.2 🎊

**Comment by tlacroix** [comment]
Not entirely fixed with 13.2 for me, but entirely fixed with 13.2.3.
